### PR TITLE
Add homepage, npm auto-update and stable versions of bokeh

### PR DESCRIPTION
**Pull request by maruilian11**
Pull request for issue: #6745

@cdnjs/intern2 Please help me review this PR, thanks!
I use git tag versions number to fetch files from official CDN url, but 0.8.0 downloaded failed.

### CODE REVIEW

File: ajax/libs/bokeh/package.json [hunk 1]
```diff
@@ -17,5 +17,15 @@
...
   },
-  "homepage": "http://bokeh.pydata.org/en/latest"
+  "homepage": "http://bokeh.pydata.org/en/latest",
+  "npmName": "bokehjs",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "js/bokeh*",
+        "css/bokeh*"
```
Comment (pvnr0082t) [inline ajax/libs/bokeh/package.json]: @maruilian11 We can make these two easier.

Comment (PeterDaveHello) [inline ajax/libs/bokeh/package.json]: Looks like we can keep the structure in this case.

**Comment by maruilian11** [comment]
@cdnjs/intern2 I updated the auto-update config.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
For the first and second commit, why don't you use single line commit message?

**Comment by maruilian11** [comment]
@cdnjs/intern2 I modify the two commits to single line, please help me review it, thanks you!

**Comment by PeterDaveHello** [comment]
@maruilian11 To be clear, the commit message of the first two commits.

**Comment by maruilian11** [comment]
updated.

**Comment by PeterDaveHello** [comment]
@maruilian11 you should not skip providing npm package info next time

**Comment by maruilian11** [comment]
got it.

**Pull request merged**